### PR TITLE
Upgrade_ABP_To_4_5_0

### PR DESCRIPTION
**Pull request by ltm0203**

### CODE REVIEW

**Comment by maliming** [comment]
Some packages are missing from the update.

File: src/AbpCompanyName.AbpProjectName.Application/AbpCompanyName.AbpProjectName.Application.csproj [hunk 1]
```diff
@@ -35,17 +35,17 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Abp, Version=4.4.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Abp.4.4.0\lib\netstandard2.0\Abp.dll</HintPath>
+    <Reference Include="Abp, Version=4.5.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Abp.4.5.0\lib\netstandard2.0\Abp.dll</HintPath>
     </Reference>
     <Reference Include="Abp.AutoMapper, Version=4.4.0.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (maliming) [inline src/AbpCompanyName.AbpProjectName.Application/AbpCompanyName.AbpProjectName.Application.csproj]: Abp.AutoMapper, Version=4.4.0.0

File: src/AbpCompanyName.AbpProjectName.Application/packages.config [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Abp" version="4.4.0" targetFramework="net461" />
+  <package id="Abp" version="4.5.0" targetFramework="net461" />
   <package id="Abp.AutoMapper" version="4.4.0" targetFramework="net461" />
```
Comment (maliming) [inline src/AbpCompanyName.AbpProjectName.Application/packages.config]: <package id="Abp.AutoMapper" version="4.4.0"

File: src/AbpCompanyName.AbpProjectName.WebApi/AbpCompanyName.AbpProjectName.WebApi.csproj [hunk 3]
```diff
@@ -35,8 +35,8 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Abp, Version=4.4.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Abp.4.4.0\lib\netstandard2.0\Abp.dll</HintPath>
+    <Reference Include="Abp, Version=4.5.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Abp.4.5.0\lib\netstandard2.0\Abp.dll</HintPath>
     </Reference>
     <Reference Include="Abp.AutoMapper, Version=4.4.0.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (maliming) [inline src/AbpCompanyName.AbpProjectName.WebApi/AbpCompanyName.AbpProjectName.WebApi.csproj]: <Reference Include="Abp.AutoMapper, Version=4.4.0.0,

File: src/AbpCompanyName.AbpProjectName.WebApi/AbpCompanyName.AbpProjectName.WebApi.csproj [hunk 4]
```diff
@@ -50,11 +50,11 @@
     <Reference Include="Abp.Web.Common, Version=4.4.0.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (maliming) [inline src/AbpCompanyName.AbpProjectName.WebApi/AbpCompanyName.AbpProjectName.WebApi.csproj]: <Reference Include="Abp.Web.Common, Version=4.4.0.0

File: src/AbpCompanyName.AbpProjectName.WebApi/packages.config [hunk 5]
```diff
@@ -1,12 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Abp" version="4.4.0" targetFramework="net461" />
+  <package id="Abp" version="4.5.0" targetFramework="net461" />
   <package id="Abp.AutoMapper" version="4.4.0" targetFramework="net461" />
```
Comment (maliming) [inline src/AbpCompanyName.AbpProjectName.WebApi/packages.config]: <package id="Abp.AutoMapper" version="4.4.0" targetFramework="net461" />
  <package id="Abp.Web" version="4.4.0" targetFramework="net461" />
  <package id="Abp.Web.Api" version="4.4.0" targetFramework="net461" />
  <package id="Abp.Web.Common" version="4.4.0" targetFramework="net461" />

**Comment by ismcagdas** [comment]
@ltm0203 thanks a lot for your contribution.

**Pull request merged**